contracts/utils/Address.sol

```diff
@@ -49,21 +79,25 @@ library Address {
      * _Available since v3.1._
      */
     function functionCall(address target, bytes memory data) internal returns (bytes memory) {
-        return functionCallWithValue(target, data, 0, "Address: low-level call failed");
+        return functionCallWithValue(target, data, 0, defaultOnRevert);
     }
 
     /**
-     * @dev Same as {xref-Address-functionCall-address-bytes-}[`functionCall`], but with
-     * `errorMessage` as a fallback revert reason when `target` reverts.
+     * @dev Same as {xref-Address-functionCall-address-bytes-}[`functionCall`], but with an
+     * `onRevert` function as a fallback when `target` reverts.
+     *
+     * Requirements:
+     *
+     * - `onRevert` must be a reverting function.
      *
      * _Available since v3.1._
      */
     function functionCall(
         address target,
         bytes memory data,
-        string memory errorMessage
+        function() internal view onRevert
```

Comment:
`onRevert` is not the most intuitive name for me here. What I find odd is that it's invoked _to_ revert, not _on_ revert.

I would go with `customRevert` instead. `defaultOnRevert` would be called `defaultRevert`.

Reply:
This makes sense to me, I updated it along with the corresponding tests and usage in Governor (`_onGovernorCallRevert` -> `_customGovernorRevert`) and SafeERC20 (`onERC20CallRevert` -> `customERC20CallRevert`)